### PR TITLE
add findBetween to k6-utils

### PR DESCRIPTION
**Pull request by tom-miseur**
This adds a simple function to k6-utils for finding a string between two strings (useful for data correlation).

### CODE REVIEW

File: lib/k6-utils/1.0.0/index.js [hunk 1]
```diff
@@ -18,4 +18,17 @@ export function randomString(length) {
   let res = '';
   while (length--) res += charset[Math.random() * charset.length | 0];
   return res;
+}
+
+function findBetween(content, left, right) {
```
Comment (MStoykov) [inline lib/k6-utils/1.0.0/index.js:23]:
```suggestion
export function findBetween(content, left, right) {
```

it needs to be exported in order to be usable

**Review by MStoykov — COMMENTED** [review]
can you also add a test in https://github.com/k6io/jslib.k6.io/blob/master/tests/basic.js

File: lib/k6-utils/1.0.0/index.js [hunk 2]
```diff
@@ -18,4 +18,17 @@ export function randomString(length) {
...
+  let start = content.indexOf(left);
+  if (start == -1) {
+    return '';
+  }
+  start += left.length;
+  const end = content.indexOf(right, start);
+  if (end == -1) {
```
Comment (sniku) [inline lib/k6-utils/1.0.0/index.js:30]: in JavaScript, it's better to use `===` for comparison like this.

**Review by sniku — CHANGES_REQUESTED** [review]
I made one small comment, but the PR should be submitted to the project repository rather than to jslib itself. 
Please make the changes and post here: https://github.com/k6io/k6-jslib-utils :smile:

**Comment by sniku** [comment]
Closing this PR since the change has been merged already in a different PR.

**Pull request closed without merging**